.evergreen/buildvariants.in.yml

```diff
@@ -34,6 +34,7 @@ buildvariants:
   - name: <% out(buildVariant.name) %>
     display_name: <% out(buildVariant.display_name) %> (Test and Package)
     run_on: <% out(buildVariant.run_on) %>
+    patchable: <% out(buildVariant.patchable) %>
```

Comment:
Should we replace run-on-pr tag with this flag in tasks now that we started to use it here? Kinda weird to have two separate mechanisms to achieve a similar behavior (like it's not exactly the same, but if we are not allowing manual patches to run these variants, it should probably be true for other tasks too)

Reply:
We could do also the opposite and use run-on-pr here, didn't figure how